src/server.c

```diff
@@ -474,26 +474,32 @@ dictType zsetDictType = {
     NULL,              /* allow to expand */
 };
 
-/* Db->dict, keys are sds strings, vals are Objects. */
-dictType dbDictType = {
+/* Kvstore->keys, keys are sds strings, vals are Objects. */
+dictType kvstoreDictType = {
```

Comment:
Include "Db" to distinguish from the other kvstore dict types.

```suggestion
dictType kvstoreDbDictType = {
```

Reply:
The wording we finalized for a `db` were `keys` and `expires`. So, this can be `kvstoreKeysDictType`.